osgbuild/constants.py

```diff
@@ -52,6 +52,9 @@
     r'^(\w*/)?osg-(?P<osgver>\d+\.\d+)$'   : 'versioned',
     r'^(\w*/)?condor-(el\d+)'              : 'condor'}
 
+MAIN_OSGVER = '3.5'
+UPCOMING_OSGVER = '3.5'
```

Comment:
Eh, so, we want to support the case where upcoming refers to a 3.X-upcoming that is not the same as the main 3.X?

Reply:
No, I just like constants.
TBH I don't expect MAIN_OSGVER to change either; maybe TRUNK_OSGVER would have been a better name but MAIN matches the SVN_RESTRICTED_BRANCHES hash.

Reply:
>No, I just like constants.

Ok, well, in that case, whichever you fancy better -- how you have it, or (for the constants lover)

```suggestion
MAIN_OSGVER = '3.5'
UPCOMING_OSGVER = MAIN_OSGVER
```